app/assets/v2/css/ideas.css

```diff
@@ -0,0 +1,63 @@
+.title {
+	text-align: center; 
+	font-family: 'Muli', sans-serif;
+}
+
+.submit-title {
+	font-size: 1.75rem
+}
+
+.justify-content-space-between {
+	display: flex; 
+	justify-content: space-between
+}
+
+.submit-text {
+	font-size: 1rem; 
+	max-width: 525px; 
+	margin: 0px auto
+}
+
+.center-button {
+	margin: 0 auto; 
+	display: block
+}
+
+.w-750px {
+	width: 750px;
+}
+
+.w-700px {
+	width: 700px;
+}
+
+#sort-ideas {
+	margin-left: 1em; 
+	width: 160px; 
+	border-style: none;
+}
+
+.idea-list {
+	width: 750px;
+	padding: 1em 1.5em; 
+	margin-bottom: 1.5em; 
+	border-style: solid; 
+	border-width: 1px; 
+	border-color: rgb(204, 204, 204); 
+	border-radius: 5px;
+	-webkit-transition: border-color 0.25s;
+    -moz-transition:    border-color 0.25s;
+    -ms-transition:     border-color 0.25s;
+    -o-transition:      border-color 0.25s;
+    transition:         border-color 0.25s;
+}
+
+.idea-list:hover {
+	border-color: purple;
```

Comment:
Wanna swap it with the hexcode instead ?
Probably pick it up from @PixelantDesign style guide